google/cloud/spanner/admin/instance_admin_connection.h

```diff
@@ -150,6 +150,33 @@ class InstanceAdminConnection {
 std::shared_ptr<InstanceAdminConnection> MakeInstanceAdminConnection(
     Options options = {});
 
+/**
+ * A factory function to construct an object of type `InstanceAdminConnection`
+ * that uses REST over HTTP as transport in lieu of gRPC. REST transport should
+ * only be used for services that do not support gRpc or if the existing network
+ * configuration precludes using gRPC.
+ *
+ * The returned connection object should not be used directly; instead it
+ * should be passed as an argument to the constructor of InstanceAdminClient.
+ *
+ * The optional @p options argument may be used to configure aspects of the
+ * returned `InstanceAdminConnection`. Expected options are any of the types in
+ * the following option lists:
+ *
+ * - `google::cloud::CommonOptionList`
+ * - `google::cloud::RestOptionList`
+ * - `google::cloud::UnifiedCredentialsOptionList`
+ * - `google::cloud::spanner_admin::InstanceAdminPolicyOptionList`
+ *
+ * @note Unexpected options will be ignored. To log unexpected options instead,
```

Comment:
FYI: s/will be ignored/are ignored/ seems simpler.  Though maybe we should make a different PR to fix all of those.